ui/package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.7.2",
     "@kubernetes/client-node": "github:scality/kubernetes-client-javascript.git#browser-0.10.2-62-g61d8543",
-    "@scality/core-ui": "github:scality/core-ui.git#3865b89",
+    "@scality/core-ui": "github:scality/core-ui.git#a0109e4",
```

Comment:
you need to use the latest commit of core-ui's development/1.0 branch

Reply:
Update the commit number of core-ui